build.gradle

```diff
@@ -840,10 +840,15 @@ task cleanAll {
 }
 task buildAll {
   dependsOn getTasksByName('assemble', true).collect { it.path }
+  dependsOn getTasksByName('product', true).collect { it.path }
```

Comment:
Running all tasks from precheckin separately works fine(e.g. ./gradlew cleanAll, buildAll, checkAll), but running the same as ./gradlew precheckin doesn't work in case of smoke bt run.  
In this case, adding product target dependency in buildAll resolved above issue.
Here, I tried to creating a separate copy task and then explicitly calling it , but it didn't work. So as a quick solution, I added above dependency.